protocol/contracts/reserve/src/state/config.rs

```diff
@@ -0,0 +1,63 @@
+use serde::{Deserialize, Serialize};
+
+use platform::contract::CodeId;
+use sdk::{
+    cosmwasm_std::Storage,
+    cw_storage_plus::Item,
+    schemars::{self, JsonSchema},
+};
+
+use crate::{api::InstantiateMsg, error::ContractResult};
+
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
+pub struct Config {
+    lease_code_id: CodeId,
+}
+
+impl Config {
+    const STORAGE: Item<'static, Self> = Item::new("config");
+
+    #[cfg(test)]
+    pub const fn new(lease_code_id: CodeId) -> Self {
```

Comment:
My IDE doesn't report there being any uses of this. I'd say you should follow your advice and remove it if it's not used.

Reply:
Update: Since the `stub` feature is for the next PR, if this is also the case here, feel free to close this.

Reply:
not anymore unused